libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -754,32 +754,32 @@ def bind_tools(
     @overload
     def with_structured_output(
         self,
-        schema: _DictOrPydanticClass,
+        schema: Optional[_DictOrPydanticClass] = None,
         *,
         method: Literal["function_calling", "json_mode"] = "function_calling",
-        return_type: Literal["all"] = "all",
+        include_raw: Literal[True] = True,
```

Comment:
is this just `: bool`

Reply:
when overloading you have to narrow it so the overloads have mutually exclusive signatures